.vscode/settings.json

```diff
@@ -44,5 +44,8 @@
   "[markdown]": {
     "editor.wordWrap": "wordWrapColumn",
     "editor.wordWrapColumn": 80
+  },
+  "[dotenv]": {
+    "editor.defaultFormatter": "foxundermoon.shell-format"
```

Comment:
Not very keen on adding more extensions to the shared settings. You can have always have more extensions active than the ones listed out in `extensions.json`.

If you feel strongly that we need to include this, please add it to the `extensions.json` file.

Reply:
Understood